src/sys/utils.cxx

```diff
@@ -61,6 +61,14 @@ const std::string lowercase(const std::string &str) {
   return strlow;
 }
 
+// Convert a string to upper case
+const std::string uppercase(const std::string& str) {
+  std::string strlow(str);
```

Comment:
Would make more sense to call this `strup` or `strupper`?

Reply:
It would -- lazy copy and paste by me there! I'll fix it.